modules/admin/src/main/resources/webapp/js/app.js

```diff
@@ -108,7 +108,7 @@
                }
             })
             .state('admin.browse', {
-               url: '/browse',
+               url: '/browse?queueName',
```

Comment:
Questions:
1. Does it correctly display the data of the **selected** queue?
2. Should this be adjusted for topics as well?